CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing <!-- omit in toc -->
+
+- [Introduction](#introduction)
+- [Ground Rules](#ground-rules)
+- [Code Contributions](#code-contributions)
+  - [Your First Code Contribution](#your-first-code-contribution)
+  - [Getting started](#getting-started)
+  - [Pull Requests](#pull-requests)
+- [Style Guides](#style-guides)
+  - [Git Commit Messages](#git-commit-messages)
+  - [Coding style](#coding-style)
+    - [Style preferences not handled by the linter](#style-preferences-not-handled-by-the-linter)
+
+# Introduction
+
+Thank you for considering contributing! It's people like you that make the open source community so great! 😊
+
+The tips and guidelines below are intended to help both contributors and maintainers respect each other's time and effect change as quickly as possible.
+
+- **Bugs and Feature Requests**: Follow the templates [here](https://github.com/melink14/rikaikun/issues/new/choose) to report problems or ideas. Don't be surprised if you're asked for clarification or opinions on solutions.
```

Comment:
> Don't be surprised if you're asked for clarification or opinions on solutions.

This comes off a little strong. Maybe something like "you may be asked to share..."

Reply:
will consider; test.

Reply:
test2